src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -133,8 +93,15 @@ def run_offline_reintegration_test(self, num_pool, data=False,
             # pver_begin + 8 targets.
             self.assertTrue(pver_exclude > (pver_begin + 8),
                             "Pool Version Error:  After exclude")
-            output = self.dmg_command.pool_reintegrate(self.pool.uuid,
-                                                       rank)
+
+        # Reintegrate the ranks which was excluded
+        for val in range(0, len(rank)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len